supplements/adventurers-league/dnd-celebration-2020.xml

```diff
@@ -0,0 +1,282 @@
+			<set name="wisdom">11</set>
+			<set name="charisma">10</set>
+			<set name="ac">13</set>
+			<set name="hp">18 (4d4 + 8)</set>
+			<set name="speed">20 ft.</set>
+			<set name="senses">blindsight 60 ft. (blind beyond this distance), passive Perception 12</set>
+			<set name="conditionImmunities">blinded, charmed, deafened, exhaustion, prone</set>
+			<set name="languages">—</set>
+			<set name="saves">Int +4, Cha +2</set>
+			<set name="type">Ooze</set>
+			<set name="size">Tiny</set>
+			<set name="alignment">lawful evil</set>
+			<set name="challenge">1/4</set>
+			<set name="traits">ID_WOTC_DNDC_COMPANION_TRAIT_GELATINOUS_ICE_CUBE_AMORPHOUS,ID_WOTC_DNDC_COMPANION_TRAIT_GELATINOUS_ICE_CUBE_AVERSION_TO_FIRE</set>
+			<set name="actions">ID_WOTC_DNDC_COMPANION_ACTION_GELATINOUS_ICE_CUBE_PSUEDOPOD</set>
+		</setters>
+		<rules>
+			<stat name="companion:ac" value="13" />
+			<stat name="companion:hp:max" value="18" bonus="base" />
+			<stat name="companion:speed" value="20" bonus="base" />
+			<stat name="companion:intelligence:save:proficiency" value="companion:proficiency" bonus="base" />
+			<stat name="companion:charisma:save:proficiency" value="companion:proficiency" bonus="base" />
+		</rules>
+	</element>
+
+	<element name="Amorphous" type="Companion Trait" source="D&amp;D Celebration 2020" id="ID_WOTC_DNDC_COMPANION_TRAIT_GELATINOUS_ICE_CUBE_AMORPHOUS">
+		<description>
+			<p>The gelatinous ice cube can move through a space as narrow as 1 inch wide without squeezing.</p>
+		</description>
+		<sheet>
+			<description>The gelatinous ice cube can move through a space as narrow as 1 inch wide without squeezing.</description>
+		</sheet>
+	</element>
+
+	<element name="Aversion to Fire" type="Companion Trait" source="D&amp;D Celebration 2020" id="ID_WOTC_DNDC_COMPANION_TRAIT_GELATINOUS_ICE_CUBE_AVERSION_TO_FIRE">
+		<description>
+			<p>If the gelatinous ice cube takes fire damage, it has disadvantage on attack rolls and ability checks until the end of its next turn.</p>
+		</description>
+		<sheet>
+			<description>If the gelatinous ice cube takes fire damage, it has disadvantage on attack rolls and ability checks until the end of its next turn.</description>
+		</sheet>
+	</element>
+
+	<element name="Psuedopod" type="Companion Action" source="D&amp;D Celebration 2020" id="ID_WOTC_DNDC_COMPANION_ACTION_GELATINOUS_ICE_CUBE_PSUEDOPOD">
+		<description>
+			<p>Melee Weapon Attack: +5 to hit, reach 5 ft., one target. Hit: 5 (1d4 + 3) bludgeoning damage plus 2 (1d4) psychic damage.</p>
+		</description>
+		<sheet>
+			<description>Melee Weapon Attack: +5 to hit, reach 5 ft., one target. Hit: 5 (1d4 + 3) bludgeoning damage plus 2 (1d4) psychic damage.</description>
+		</sheet>
+	</element>
+	<!--/Gelatinous Ice Cube-->
+
+	<!--Snowy Owlbear Cub-->
+	<element name="Snowy Owlbear Cub" type="Companion" source="D&amp;D Celebration 2020" id="ID_WOTC_DNDC_COMPANION_SNOWY_OWLBEAR_CUB">
+		<supports>Familiar</supports>
```

Comment:
Due to it being non-standard PHB familiar, it might be better if we keep it as an optional Familiar. We maybe should talk about it.
```suggestion
		<supports>Variant Familiar</supports>
```